### PR TITLE
Fix the issue on CampcotCollectionView github repository 

### PR DESCRIPTION
**Pull request by AlexYanski**
Added storyboard support.

### CODE REVIEW

File: Example/Source/StoryboardViewController.swift [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+class StoryboardViewController: UIViewController {
+
+    @IBOutlet weak var collectionView: CampcotCollectionView!
+
+    let itemsInRow = 2
+    var itemsInSection: [Int: Int] = [:]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.collectionView.delegate = self
```
Comment (pkandrashkou) [inline Example/Source/StoryboardViewController.swift]: You can set this up through storyboard too. dataSource is the same.

File: Example/Source/StoryboardViewController.swift [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        self.collectionView.dataSource = self
+        self.collectionView.translatesAutoresizingMaskIntoConstraints = false
```
Comment (pkandrashkou) [inline Example/Source/StoryboardViewController.swift]: this is not needed

File: README.md [hunk 3]
```diff
@@ -59,6 +61,13 @@ public func toggle(to section: Int,
                    animated: Bool,
                    completion: ((Bool) -> Void)? = default)
 ```
+### Storyboard Setup
+
+1. Add UICollectionView to your Storyboard.
+2. In `Identity Inspector` set the `Class` property to `CampcotCollectionView`.
+3. Open `Attributes Inspector` and set the `Layout` property to `Custom`. Bellow set `Class` property to either `ExpandedLayout` or `CollapsedLayout`.
```
Comment (pkandrashkou) [inline README.md]: Remove "Bellow" word :)

File: README.md [hunk 4]
```diff
@@ -59,6 +61,13 @@ public func toggle(to section: Int,
...
+4. Create outlet for your collectionView.
```
Comment (pkandrashkou) [inline README.md:69]: Mention "Set datasource and delegate" too :)

File: Source/CampcotCollectionView.swift [hunk 5]
```diff
@@ -46,9 +49,38 @@ public class CampcotCollectionView: UICollectionView {
             self.collapsedLayout.sectionInset = sectionInset
         }
     }
-    
+
+    @IBInspectable
+    private var topInset: CGFloat = 0 {
+        didSet {
+            sectionInset.top = topInset
```
Comment (pkandrashkou) [inline Source/CampcotCollectionView.swift:56]: Do these "did set"s work?

File: README.md (hunk 3 above)
Comment (AlexYanski) [reply README.md]: Ooops. Fixed :)

File: Example/Source/StoryboardViewController.swift (hunk 2 above)
Comment (AlexYanski) [reply Example/Source/StoryboardViewController.swift]: Done

File: Example/Source/StoryboardViewController.swift (hunk 1 above)
Comment (AlexYanski) [reply Example/Source/StoryboardViewController.swift]: Done

File: README.md (hunk 4 above)
Comment (AlexYanski) [reply README.md:69]: Done

File: Source/CampcotCollectionView.swift (hunk 5 above)
Comment (AlexYanski) [reply Source/CampcotCollectionView.swift:56]: Yes, it is working

**Review by pkandrashkou — APPROVED** [review]

**Pull request merged**